client/src/components/sideBar/SideBar.js

```diff
@@ -89,13 +95,19 @@ export function SideBar({ shouldHide, isLoading }) {
         label='Light Mode'
         onClick={() => {
           dispatch(toggleLightMode())
+          dispatch(resetView())
```

Comment:
small thing,

do we want to keep resetting the map to the globe view between light/dark? 

e.g.: 
say I have a Trip to Vancouver open with see more drawer/compact view and I decide to turn on light mode. then currently it resets all the views.

Reply:
good note ☃️ tbh I also prefer if we don't reset it. however an issue is, when the map is changing, mapbox sometimes randomly throws errors bc it's trying to add markers to the map while the map is changing to light mode. 

I wasn't able to figure that out in a reasonable amount of time, so I added this bandaid fix. open to discussing ideas / adding a backlog to fix it tho ^^

Reply:
update: this is highly mitigated, with
1. not re-making the map when light mode changes (instead opting to change it in a separate useeffect)
2. try/catch wherever code is synchronous (although there's many async calls within mapbox's rendering logic itself, so i can't guarantee 0 runtime errors. js only catches synchronous runtime errors)
3. a 1-second delay between toggle clicks, because this problem often comes up during rapid color toggling.

Outcomes: the map won't reset back to globe view anymore, so you can continue where you left off.

Reply:
However as noted in 2, mapbox's rendering logic isn't something we can control. It may still call a chain of async calls that throw runtime render errors, given the right sequence of events. But due to the mitigations mentioned above, this is likely very sparse now (I couldn't reproduce it during testing). 

The errors were always non-blocking anyways. The app wouldn't crash, and everything looks normal (likely because mapbox was just trying to render a pixel that was no longer within view).

Reply:
Sorry for all the text. I mainly wanted to document this in case a TA came across the issue, then we have a paper trail explaining the cause.